src/lib/mongoBackend/mongoConnectionPool.cpp

```diff
@@ -191,9 +193,14 @@ static DBClientBase* mongoConnect
 
   if (connected == false)
   {
-    LM_E(("Database Error (connection failed, after %d retries: '%s')", retries, err.c_str()));
+    char cV[64];
```

Comment:
I would prefer to keep the 64 or whatever there is.

I believe these are pretty much unique cases ...